governance/default.go

```diff
@@ -550,44 +566,8 @@ func (gov *Governance) updateChangeSet(vote GovernanceVote) bool {
 	return false
 }
 
-func GetDefaultGovernanceConfig(engine params.EngineType) *params.GovernanceConfig {
```

Comment:
Can I ask why you move those getter functions(GetDefault~~) from governance package to params package?

Reply:
`GovernanceConfig` struct is defined in `params` package and `GetDefaultGovernanceConfig` also used variables defined in `params` package, but not in `governance` package. As `GetDefaultGovernanceConfig` is located in `params` package, other packages calling `GetDefaultGovernanceConfig` should import `governance` package as well as `params` package. It increases the complexity of the import chain and may suffer unnecessary package cycle issues later.